remote-admin-create-cache/README.adoc

```diff
@@ -0,0 +1,23 @@
+= Remote Cache Administration
+
+**Authors:** Wolf Fink +
+**Technologies:** Infinispan, Hot Rod, Java +
+**Summary:** Use Hot Rod Java clients to remotely create and administer caches
+on Infinispan servers.
+
+. Open `infinispan.xml` for editing under `server/conf` in the server installation directory.
+. Add the following cache configuration to create a template:
++
+----
+<distributed-cache-configuration name="MyDistCachecConfig" mode="ASYNC">
```

Comment:
can't we propose a docker option so the infinispan.xml is already provided and we run the docker image with a command line?

Reply:
I would not involve docker container here as it is another story

Reply:
but this means for the tutorial people need to download the server, change by hand the xml file and run it. it's very long and should be IMO an option. A single command line should be provided so the users can run it fast (in the readme)

Reply:
That said, I'm not going to block this for the merge. it can be done afterwards in another PR (the X-Site tutorial should be in too, and I can do another commit with the documentation on how to make it work in a single docker command line)